querydsl-bom/pom.xml

```diff
@@ -16,6 +16,10 @@
     <url>${project.homepage}</url>
     <packaging>pom</packaging>
 
+    <properties>
+        <flatten-maven-plugin.version>1.2.7</flatten-maven-plugin.version>
```

Comment:
This property shows up in the generated pom. Is that necessary?

Reply:
Removed it from flattened pom.